community/pom.xml

```diff
@@ -0,0 +1,376 @@
+							<includes>resources/widgets/custom*/**</includes>
+							<outputDirectory>${web.widgets.dir}</outputDirectory>
+							<overWriteReleases>true</overWriteReleases>
+							<overWriteSnapshots>true</overWriteSnapshots>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<artifactId>maven-assembly-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>bos</id>
+						<configuration>
+							<descriptors>
+								<descriptor>bos-assembly.xml</descriptor>
+							</descriptors>
+							<appendAssemblyId>true</appendAssemblyId>
+							<attach>false</attach>
+						</configuration>
+						<phase>package</phase>
+						<goals>
+							<goal>single</goal>
+						</goals>
+					</execution>
+					<execution>
+						<id>zip</id>
+						<configuration>
+							<descriptors>
+								<descriptor>zip-assembly.xml</descriptor>
+							</descriptors>
+							<appendAssemblyId>false</appendAssemblyId>
+							<attach>false</attach>
+						</configuration>
+						<phase>package</phase>
+						<goals>
+							<goal>single</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>com.coderplus.maven.plugins</groupId>
+				<artifactId>copy-rename-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>rename-to-bos-extension</id>
+						<phase>package</phase>
+						<configuration>
+							<sourceFile>${project.build.directory}/${project.artifactId}-${project.version}-bos.zip</sourceFile>
+							<destinationFile>${project.build.directory}/${project.artifactId}-${project.version}.bos</destinationFile>
+							<overWrite>true</overWrite>
+						</configuration>
+						<goals>
+							<goal>rename</goal>
+						</goals>
+					</execution>
+					<execution>
+						<id>prepare-xml-artifact</id>
+						<phase>package</phase>
+						<configuration>
+							<sourceFile>bonita-distrib/${project.artifactId}.xml</sourceFile>
+							<destinationFile>${project.build.directory}/${project.artifactId}-${project.version}.xml</destinationFile>
+							<overWrite>true</overWrite>
+						</configuration>
+						<goals>
+							<goal>copy</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>build-helper-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>attach-artifact</goal>
+						</goals>
+						<configuration>
+							<artifacts>
+								<artifact>
+									<type>bos</type>
+									<file>${project.build.directory}/${project.artifactId}-${project.version}.bos</file>
+								</artifact>
+								<artifact>
+									<type>zip</type>
+									<file>${project.build.directory}/${project.artifactId}-${project.version}.zip</file>
+								</artifact>
+								<artifact>
+									<type>xml</type>
+									<file>${project.build.directory}/${project.artifactId}-${project.version}.xml</file>
```

Comment:
Will we remove that once it's not used by the `distrib` project anymore?

Reply:
I wonder.
Maybe once we finish the work in the distrib and we make sure we don't need it